Queueomatic/Server/Endpoints/User/Delete/DeleteUserEndpoint.cs

```diff
@@ -1,27 +1,35 @@
 ﻿using FastEndpoints;
+using Queueomatic.DataAccess.UnitOfWork;
+using Queueomatic.Server.Services.AuthenticationService;
 
 namespace Queueomatic.Server.Endpoints.User.Delete;
 
-public class DeleteUserEndpoint: Endpoint<DeleteUserRequest, DeleteUserResponse>
+public class DeleteUserEndpoint: Endpoint<DeleteUserRequest>
 {
+    private readonly IUnitOfWork _unitOfWork;
+
+    public DeleteUserEndpoint(IUnitOfWork unitOfWork)
+    {
+        _unitOfWork = unitOfWork;
+    }
     public override void Configure()
     {
         Verbs(Http.DELETE);
         Routes("/users/{email}");
+        Policies("ValidParticipant");
```

Comment:
This is the wrong policy. Please use the one for users.

Reply:
Solved